### PR TITLE
fix: add underscored in db config

### PR DESCRIPTION
**Pull request by ijas-ws**
### Description

I have changed the 'userscored' to 'underscored'. 

---

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -11,12 +11,12 @@ services:
     image: 'redis:alpine'
     ports:
       - '6379:6379'
-    command: ['redis-server', '--bind', 'redis', '--port', '$REDIS_PORT']
+    command: ['redis-server', '--bind', 'redis', '--port', '6379']
   app:
     build:
       context: .
       args:
-        ENVIRONMENT_NAME: ${ENVIRONMENT_NAME}
```
Comment (alichherawalla) [inline docker-compose.yml]: we should be able to infer these values by passing an environment file to it

Comment (ijas-ws) [reply docker-compose.yml]: Okey. resolved it.

File: docker-compose.yml [hunk 2]
```diff
@@ -11,7 +11,7 @@ services:
     image: 'redis:alpine'
     ports:
       - '6379:6379'
-    command: ['redis-server', '--bind', 'redis', '--port', '$REDIS_PORT']
+    command: ['redis-server', '--bind', 'redis', '--port', '6379']
```
Comment (alichherawalla) [inline docker-compose.yml]: same for this right. We shouldn't need to replace the ENVIRONMENT VARIABLE

Comment (ijas-ws) [reply docker-compose.yml]: We are already telling that we are using 6379 in the above(13th) line, then why should we add the env value in 14th line. I didn't understand that.

Comment (alichherawalla) [reply docker-compose.yml]: we should change the outbound port based on the env variable on line 13 as well

**Review by alichherawalla — APPROVED** [review]

**Pull request merged**